import/index_java/src/org/vufind/index/ConfigManager.java

```diff
@@ -210,55 +221,59 @@ public Ini loadConfigFile(String filename)
                 retVal.put(key, overrideSection.get(key));
             }
         }
+
+        if (retVal == null) {
+            logger.warn(section + " section missing from " + filename);
+        }
         return retVal;
     }
 
     /**
-     * Get a setting from a VuFind configuration file.
+     * @deprecated
+     */
+    public Map<String, String> getSanitizedConfigSection(String filename, String section)
+    {
+        return getConfigSection(filename, section);
+    }
+
+    /**
+     * Get a setting from a VuFind configuration file and sanitize the value.
      * @param filename configuration file name
      * @param section section name within the file
      * @param setting setting name within the section
      */
     public String getConfigSetting(String filename, String section, String setting)
     {
-        String retVal = null;
-
-        // Grab the ini file.
-        Ini ini = loadConfigFile(filename);
+        String retVal = getRawConfigSetting(filename, section, setting);
+        return retVal == null ? retVal : sanitizeConfigSetting(retVal);
+    }
 
-        // Check to see if we need to worry about an override file:
-        String override = ini.get("Extra_Config", "local_overrides");
-        if (override != null) {
-            Ini overrideIni = loadConfigFile(override);
-            retVal = overrideIni.get(section, setting);
-            if (retVal != null) {
-                return sanitizeConfigSetting(retVal);
-            }
+    /**
+     * Get a setting from a VuFind configuration file.
+     * @param filename configuration file name
+     * @param section section name within the file
+     * @param setting setting name within the section
+     */
+    public String getRawConfigSetting(String filename, String section, String setting)
+    {
+        Map<String, String> sectionMap = getConfigSection(filename, section);
```

Comment:
Now that getConfigSection returns sanitized values, this is going to prevent getRawConfigSetting from returning raw config settings. I assume you want to use getRawConfigSection instead.